test/integration/manifests/goldpinger/daemonset-ipv6.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
```

Comment:
can we use k8s network conformance test for linux dualstack instead of goldpinger?

Reply:
yes